src/main/kotlin/com/ebay/plugins/metrics/develocity/DevelocityConfigurationInputs.kt

```diff
@@ -0,0 +1,22 @@
+package com.ebay.plugins.metrics.develocity
+
+import org.gradle.api.Task
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+
+/**
+ * Interface which may be added to a task to have it be auto-configured with the Develocity
+ * server configuration as applied to the root project in the [MetricsForDevelocityExtension].
+ *
+ * NOTE: Access to the Develosity access key is intentionally not provided here, as it is
```

Comment:
typo:
Develosity -> Develocity
```suggestion
 * NOTE: Access to the Develocity access key is intentionally not provided here, as it is
```